sidebars.js

```diff
@@ -683,6 +683,11 @@ const sidebars = {
         id: "articles/examples/overview",
       },
       items: [
+                {
+                  type: "doc",
+                  label: "Custom Test Workflow Templates",
```

Comment:
sorry - last comment - please 
- move this to be under the Workflows Cookbook section in the menu
- name it "Creating your own Workflow Templates"
- add links between this and the doc at https://docs.testkube.io/articles/test-workflow-templates